Containerfile

```diff
@@ -0,0 +1,25 @@
+# fedora-36 
+FROM registry.fedoraproject.org/fedora:36 
+
+WORKDIR /vagrant
+
+COPY requirements.txt /vagrant
+
+RUN \
+    dnf -y install \
+#      python3 \
+      sqlite \
+      pip \
+      gcc \
+      python3-devel \
+    && \
+    dnf clean all
+
+RUN \    
+    python3 -m pip install -r requirements.txt
```

Comment:
please build a version that uses pipenv - requirements.txt is not an adequate replacement.

try using Red Hat's Python s2i image.

Reply:
I see this
https://github.com/sclorg/s2i-python-container/blob/master/3.6/README.md
It is very interesting but not clear for me, 

I see an option for django pipenv and for release gunicorn option,
Could be that i need more guidance here

Reply:
Found this https://developers.redhat.com/articles/2022/11/22/pipenv-and-s2i-better-way-manage-python-dependencies-containers#shortcomings_of_pip